lib/bacnet-asn1.js

```diff
@@ -1724,29 +1720,57 @@ var bacapp_decode_context_data = function(buffer, offset, maxApduLen, propertyTa
   };
 };
 
-var encode_bacnet_character_string = function(buffer, value) {
-  buffer.buffer[buffer.offset++] = baEnum.BacnetCharacterStringEncodings.CHARACTER_UTF8;
-  buffer.offset += buffer.buffer.write(value, buffer.offset, undefined, 'utf8');
+var encode_bacnet_character_string = function(buffer, value, encoding) {
```

Comment:
Doesn't the `get_node_encoding` function already falls back to `utf8` anyway as default value? Can't we assume that encoding always is defined?